lib/rspec/core/option_parser.rb

```diff
@@ -291,6 +291,8 @@ def initialize_project_and_exit
     def bisect_and_exit(argument)
       RSpec::Support.require_rspec_core "bisect/coordinator"
 
+      # This might be the best spot to mutate ENV['SPEC_OPTS']
+      # if we do that.
```

Comment:
If I'm reading code correctly, I can handle the env vars here, or multiple times in each runner.

B mobile phone

> On Jun 4, 2016, at 12:47 PM, Myron Marston <notifications@github.com> wrote:
> 
> In lib/rspec/core/option_parser.rb:
> 
> > @@ -291,6 +291,8 @@ def initialize_project_and_exit
> >      def bisect_and_exit(argument)
> >        RSpec::Support.require_rspec_core "bisect/coordinator"
> >  
> > +      # This might be the best spot to mutate ENV['SPEC_OPTS']
> > +      # if we do that.
> It's not obvious to me what's better about this spot then the spot you already have it. Can you explain?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>